<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregar</class>
 <widget class="QDialog" name="agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>230</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AGREGAR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/imagenes/ICONOS/agregar-simbolo.png</normaloff>:/imagenes/ICONOS/agregar-simbolo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(69, 105, 127);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 127);</string>
     </property>
     <property name="title">
      <string>NUEVA TAREA</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string>color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Nombre de la Tarea:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="InNombre">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(217, 255, 249);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string>color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Fecha Maxima:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QDateEdit" name="Infecha">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(217, 255, 249);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string>color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Prioridad:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="inPrioridad">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(217, 255, 249);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string>color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Tipo:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="inTipo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(217, 255, 249);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>186</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="checkFinalizado">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Finalizado</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>61</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnAgregar">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(198, 255, 172);</string>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QPushButton" name="btnCancelar">
        <property name="font">
         <font>
          <family>Georgia</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 176, 201);</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
